ckan/config/middleware/flask_app.py

```diff
@@ -360,3 +379,16 @@ def is_blueprint(mm):
         for blueprint in inspect.getmembers(module, is_blueprint):
             app.register_blueprint(blueprint[1])
             log.debug(u'Registered core blueprint: {0!r}'.format(blueprint[0]))
+            for rule in app.url_map.iter_rules():
```

Comment:
After looking at the code it's now clear to me as well that we can't avoid `config['routes.named_routes']` until we just have Flask endpoints. So let's keep this, but I think it should be moved to another place where it only gets run once and it also cover extension. A good spot seems to be [here](https://github.com/tino097/ckan/blob/8d8c222c71a9d4c6be2d1f9e8613ebfc2286363c/ckan/config/middleware/flask_app.py#L157) @smotornyuk WDYT?